google-cloud-logging/src/main/java/com/google/cloud/logging/CmekSettings.java

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.logging;
+
+import com.google.common.base.Function;
+import com.google.common.base.MoreObjects;
+import java.util.Objects;
+
+/**
+ * CmekSettings Describes the customer-managed encryption key (CMEK) settings associated with a
+ * project, folder, organization, billing account, or flexible resource.
+ *
+ * <p>Note: CMEK for the Logs Router can currently only be configured for GCP organizations. Once
+ * configured, it applies to all projects and folders in the GCP organization.
+ *
```

Comment:
'can currently only be configured for' -> 'currently can be configured only for'